101-backup-vault-basic/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vaultName": {
```

Comment:
remove this when you add the defaultValue

Reply:
removed this.